### ISSUE TITLE
Publishing does not work completely

### ISSUE THREAD

**Issue by Cyb0rk** [rtconner/laravel-tagging#46]
When I try to publish it only copies the config.php.

```
XXX@XXX:~/XXX$ php artisan vendor:publish --provider="Conner\Tagging\TaggingServiceProvider"
Copied File [/vendor/rtconner/laravel-tagging/config/tagging.php] To [/config/tagging.php]
Publishing Complete!
```

I had to copy the migration files per hand.

**Comment by rtconner** [comment]
Dangit there was a bug in the migration publish. This is what I get for merging pull requests without testing. Thanks for the report.

**Comment by rtconner** [comment]
Fixed in version 1.0.7
